src/components/content/order/migrate/MigratingServices.tsx

```diff
@@ -0,0 +1,375 @@
+                    deploying={deploying}
+                    formRef={formRef}
+                />
+            ),
+            description: 'Deploy the service.',
+        },
+        {
+            title: 'Import Data',
+            content: <ImportServiceData />,
+            description: 'Import service data.',
+        },
+        {
+            title: 'Destroy',
+            content: (
+                <DestroyService
+                    userAvailableServiceVoList={userAvailableServiceVoList}
+                    selectCsp={selectCsp}
+                    selectArea={selectArea}
+                    selectRegion={selectRegion}
+                    selectFlavor={selectFlavor}
+                />
+            ),
+            description: 'Destroy the service.',
+        },
+    ];
+
+    const items = steps.map((item) => ({ key: item.title, title: item.title, description: item.description }));
+
+    return (
+        <div className={'migrate-select-destination-class'}>
+            {requestSubmitted || deploying || destroying ? tip : <></>}
+            <div className={'migrate-step-title-inner-class'}>
+                <Steps current={current} items={items} />
+            </div>
+            <div className={'migrate-step-content-inner-class'}>{steps[current].content}</div>
+            <div className={'migrate-step-button-inner-class'}>
+                <Space size={'large'}>
+                    {current < steps.length - 1 && (
+                        <Button
+                            type='primary'
+                            style={{ width: '85px' }}
+                            onClick={() => next()}
+                            disabled={requestSubmitted}
+                        >
+                            Next
+                        </Button>
+                    )}
+
+                    {current > 0 && (
+                        <Button
+                            type='primary'
+                            style={{ width: '85px' }}
+                            onClick={() => prev()}
+                            disabled={requestSubmitted}
+                        >
+                            Previous
+                        </Button>
+                    )}
+
+                    {current === steps.length - 3 && (
+                        <Button
+                            type='primary'
+                            style={{ width: '85px' }}
+                            htmlType='submit'
+                            onClick={deploy}
+                            disabled={deploying || requestSubmitted || destroying}
+                        >
+                            Deploy
+                        </Button>
+                    )}
+                    {current === steps.length - 1 && (
+                        <Popconfirm
+                            title='Destroy the migrated service'
+                            description='Are you sure to destroy the migrated service?'
```

Comment:
`Are you sure to destroy the old service?`

Reply:
fixed